.dockerignore

```diff
@@ -0,0 +1,6 @@
+.git
+.gitignore
+.dockerignore
+vendor
```

Comment:
Why do you ignore the vendor folder and redownload all dependencies on every build?

Reply:
Fetching dependencies usually is only cheap if you have the complete control over your build environment. Using vendored dependencies improved reliability of builds and reproducibility. For instance, if Github is down or an upstream gets renamed or removed, we can still release without having to work around that :)

Reply:
Builds are reliable and reproducible as you have locked dependencies (there's a reference to commit).

I'm concerned about security aspect of vendor directory. It may contain modified sources.